Makefile

```diff
@@ -23,16 +23,15 @@ test:
 	cram test
 
 install:
-	install -Dm755 downgrade $(DESTDIR)/$(PREFIX)/bin/downgrade
-	install -Dm644 doc/downgrade.8 $(DESTDIR)/$(MANPREFIX)/man8/downgrade.8
-	install -Dm644 completion/bash $(DESTDIR)/$(PREFIX)/share/bash-completion/completions/downgrade
-	install -Dm644 completion/zsh $(DESTDIR)/$(PREFIX)/share/zsh/site-functions/_downgrade
+	install -Dm755 downgrade $(DESTDIR)$(PREFIX)/bin/downgrade
+	install -Dm644 doc/downgrade.8 $(DESTDIR)$(MANPREFIX)/man8/downgrade.8
+	install -Dm644 completion/bash $(DESTDIR)$(PREFIX)/share/bash-completion/completions/downgrade
+	install -Dm644 completion/zsh $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_downgrade
 
 uninstall:
-	$(RM) $(DESTDIR)/$(PREFIX)/bin/downgrade \
-	  $(DESTDIR)/$(MANPREFIX)/man8/downgrade.8 \
-	  $(DESTDIR)/$(PREFIX)/share/bash-completion/completions/downgrade \
-	  $(DESTDIR)/$(PREFIX)/share/zsh/site-functions/_downgrade
+	$(RM) $(DESTDIR)$(PREFIX)/bin/downgrade \
+	  $(DESTDIR)$(MANPREFIX)/man8/downgrade.8 \
+	  $(DESTDIR)$(PREFIX)/share/bash-completion/completions/downgrade
```

Comment:
I agree with the idea here to make it cleaner. But the `PREFIX` defined in this `Makefile` has no trailing slash symbol so there is an inconsistency between `DESTDIR` and `PREFIX`. If we want to keep this, `PREFIX` should be `/usr/local/` and line 26 (as an example) would become `install -Dm755 downgrade $(DESTDIR)$(PREFIX)bin/downgrade`.

I would be uncomfortable with this version because if a trailing slash is erroneously forgotten during say manual installation, it would lead to a wrong prefix and wrong (perhaps non-existant) installation path. However, if we leave the structure as before and allow for repeating slashes (even when unnecessary), we can be sure that this error would never arise and `downgrade` will always be installed to the correct (intended) location.

Reply:
> Path variables should have leading slashes themselves (like the default
PREFIX), so $(DESTDIR)/$(PREFIX) was actually "//usr/local". This
doesn't matter, but is unnecessary.

Ah just read this commit message. Hmm, yes if we are sure the `DESTDIR` will always have a leading slash this should be fine. But if want to play it safe we could keep the slash symbol between `DESTDIR` and `PREFIX`. Let me know your thoughts :)

Reply:
Nice catch! I'm not sure what is least surprising to a packager. I think it's worth enumerating the cases (and possibly writing a small cram test for it?):

What do you think of these outcomes, and can you list any more interesting combinations?

```
make DESTDIR= PREFIX= install
# should install bin/downgrade or /bin/downgrade?

make DESTDIR=/d PREFIX= install
# should install /d/bin/downgrade

make DESTDIR=d PREFIX= install
# should install d/bin/downgrade

make DESTDIR= PREFIX=/p install
# should install /p/bin/downgrade

make DESTDIR= PREFIX=p install
# should install p/bin/downgrade

make DESTDIR=/d PREFIX=/p install
# should install /d/p/bin/downgrade

make DESTDIR=d PREFIX=/p install
# should install d/p/bin/downgrade

make DESTDIR=d PREFIX=p install
# will install dp/bin/downgrade
# sorrynotsorry
```

> I would be uncomfortable with this version because if a trailing slash is erroneously forgotten...

FWIW, since this is a _packager_ interface and not a user interface (yes, someone installing from source plays both roles at different times, but they're still different roles), I don't mind there being risks if you get the slashes wrong. The complexity of making it totally free of foot-guns isn't there IMO. So if we come up with a nice simple approach that solves the cases above, but maybe could be misused accidentally somehow, that'd be fine with me.

Reply:
> Let me know your thoughts :)

I'm going to keep this as-is for now, but I'm not against revisiting in a separate PR and improving things.

Reply:
Good morning :sun_behind_small_cloud:

Did not have enough time to think about this yesterday. I agree that this is minor and we can discuss this in a future PR if need be.